component-test/test_dbus_proxy.py

```diff
@@ -275,13 +267,21 @@ def test_directory_whitelist(self, session_bus, service_on_outside, dbus_proxy,
         assert "My unique key" in captured_stdout
 
     @pytest.mark.parametrize("config", [
-        CONF_WHITELIST_PATH,
-        CONF_WHITELIST_PATH_REVERSE
+        CONF_CONTRADICTING_RULES,
+        CONF_CONTRADICTING_RULES_DIFFERENT_ORDER
```

Comment:
I think it looks like this is essentially like above but for interfaces. If so, I think the configs should be named in a manner that looks like the rest, and that the tests should be part of the parametrized "whitelisting" tests above.